pkg/instance/repository.go

```diff
@@ -48,10 +46,13 @@ func (r repository) FindWithParametersById(id uint) (*model.Instance, error) {
 	return instance, err
 }
 
-func (r repository) FindByNameAndGroup(instanceName string, groupId uint) (*model.Instance, error) {
+func (r repository) FindByNameAndGroup(instanceName string, groupName string) (*model.Instance, error) {
```

Comment:
If there is no name collision I would name the args `instance` and `group`. The method says FindByName. Name is also now the identifier for almost all things. In an ambiguous context we can postfix the `id` or `name`. Its clear here